backend/corpora/lambdas/api/v1/collection_uuid/upload.py

```diff
@@ -24,6 +24,8 @@ def link(collection_uuid: str, body: dict, user: str):
         resp = dropbox.get_file_info(url)
     except requests.HTTPError:
         raise InvalidParametersHTTPException("The URL provided causes an error with Dropbox.")
+    except KeyError:
```

Comment:
Could you make this a more explicit exception, like maybe a `dropbox.MissingHeaderException`?

Reply:
I can, but I'm not really sure How I would test this.

Reply:
I can try mocking the requests library to return a response without headers.